docs/server/commands.rst

```diff
@@ -581,6 +581,50 @@ and the process will start a watchdog to track any client-side scripts for
 changes. Use this only when developing Pootle.
 
 
+.. _commands#user-management:
+
+Managing users
+--------------
+
+.. django-admin:: merge_user
+
+merge_user
+^^^^^^^^^^
+
+.. versionadded:: 2.7.1
+
+If you have a user with two accounts or need to merge one account into another
+you can use `merge_user`. This will re-assign all submission, units and
```

Comment:
Release notes are mainly to leave users know about changes, so providing a quick reasoning why these commands where introduced would be really good. For example `purge_user` is to kick evil users :)